crates/bevy_pbr/src/light_probe/mod.rs

```diff
@@ -145,6 +145,9 @@ where
     // The transform from world space to light probe space.
     light_from_world: Affine3A,
 
+    // The transpose of the inverse of [`light_from_world`].
+    light_from_world_transposed: Mat4,
```

Comment:
If we are to improve performance, this would need to replace `light_from_world` with the three axes used in `maybe_gather_light_probes`. `light_from_world` otherwise goes completely unused.